src/accountant/core.cljs

```diff
@@ -83,7 +83,7 @@
            host (.getDomain uri)
            current-host js/window.location.hostname]
        (when (and (not any-key) (= button 0) (path-exists? path) (= host current-host))
-         (set-token! history relative-href title)
+         (. history (setToken relative-href title))
```

Comment:
Where does this function come from? I'd prefer that we clearly namespace this sort of thing.